great-idea/css/index.css

```diff
@@ -63,4 +63,232 @@ h1, h2, h3, h4, h5 {
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* header */
+
+header nav {
+  display: flex;
+  flex-direction: row;
+  width: 691px;
+  vertical-align: top;
+  margin-top: 35px;
+    
+}
+
+header nav a {
+  display: flex;
+  flex-direction: row;
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: flex;
+  position: relative;
+    right: 18px;
+    bottom: 30px;
+  margin-bottom: 0px;
+  padding-left: 200px;
+  float: right;
+
+}
+
+/* cta */
+
+.cta .cta-text {
+    flex-direction: row;
+    font-size: 77px;
+    margin-top: 120px;
+    text-align: center;
+    padding: 0px 75px;
+    padding-right: 560px;
+}
+
+.cta .cta-text button {
+    background: white;
+    vertical-align: top;
+    margin-top: 6px;
+    padding: 8px 44px;
+    font-size: 17px;
+    border: 1px solid black;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+
+}
+
+.cta img {
+    display: flex;
+    float: right;
+    position: relative;
+    top: -350px;
+
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 340px;
+  padding-top: 0px;
+  position: relative;
+  top: -240px;
+  
+
+}
+
+.main-content .top-content {
+    padding: 0 10px;
+    float: right;
+
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  flex-direction: row;
+  margin-right: 34px;
+  float: left;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+
+  
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  padding: 0 10px;
+  flex-direction: row;
+
+}
+
+.main-content .bottom-content .text-content {
+    width: 250px;
+    flex-direction: row;
+    margin-right: 34px;
+    float: left;
+    justify-content: center;
+   padding-top: 39px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+    border-top: 1px solid black;
+  margin-top: 10px;
+  padding: 39px 0px 0;
+  position: relative;
+  bottom: 20px;
+
+}
+.contact h4 { 
+    position: relative;
+  top: -130px;
+}
+
+.contact p {
+  margin: 20px 0;
+  position: relative;
+  bottom: 135px;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
```

Comment:
Your code looks really clean in this project and I like how specific you were with a lot of your callouts and commenting sections out.. keep up the great work, you're doing awesome.